### PR TITLE
raise pandas EmptyDataError when wks.get_all_values() returns None

### PR DESCRIPTION
**Pull request by gregroberts**
Fixes /maybelinot/df2gspread#42

### CODE REVIEW

File: df2gspread/gspread2df.py [hunk 1]
```diff
@@ -82,7 +82,9 @@ def download(gfile, wks_name=None, col_names=False, row_names=False,
     raw_data = wks.get_all_values()
 
     if not raw_data:
```
Comment (maybelinot) [inline df2gspread/gspread2df.py]: `raw_data` is not a pandas object, so I'd go with python `ValueError` here

**Comment by maybelinot** [comment]
Thank you for the PR @gregroberts

File: df2gspread/gspread2df.py (hunk 1 above)
Comment (gregroberts) [reply df2gspread/gspread2df.py]: Very valid point. Amended and pushed.

**Comment by maybelinot** [comment]
I'm going to merge it now and bump the package version after data type fixes.
Thank you for the contribution! :slightly_smiling_face:

**Comment by gregroberts** [comment]
Cheers for the opportunity to contribute! I really really like this library, and it fills a really big gap in our ETL process.

**Pull request merged**